### PR TITLE
Connect wal to test

### PR DESCRIPTION
**Pull request by tock-ibm**

### CODE REVIEW

File: test/test_app.go [hunk 1]
```diff
@@ -212,3 +217,45 @@ func newNode(id uint64, network Network, testName string) *App {
 	app.Node = network[id]
 	return app
 }
+
+func initWAL(logger api.Logger, walDir string) (consensusWAL *wal.WriteAheadLogFile, walInitState [][]byte) {
```
Comment (yacovm) [inline test/test_app.go:280]: Is this logic equivalent to what is in the fabric repository? Can't we move this to a separate utility function in the WAL package?

File: test/basic_test.go [hunk 2]
```diff
@@ -47,10 +53,14 @@ func TestRestartFollowers(t *testing.T) {
 	network := make(Network)
 	defer network.Shutdown()
 
-	n1 := newNode(1, network, t.Name())
-	n2 := newNode(2, network, t.Name())
-	n3 := newNode(3, network, t.Name())
-	n4 := newNode(4, network, t.Name())
+	testDir, err := ioutil.TempDir("", "test-restart-followers")
```
Comment (yacovm) [inline test/basic_test.go]: you can use t.Name() in all of the tests instead explicitly specifying the test name, no?

File: test/test_app.go (hunk 1 above)
Comment (tock-ibm) [reply test/test_app.go:280]: Yes it is, my plan is to do it a separate PR in which I also change the Fabric repo.

File: test/basic_test.go (hunk 2 above)
Comment (tock-ibm) [reply test/basic_test.go]: good idea!

**Review by yacovm — APPROVED** [review]

**Pull request merged**